### ISSUE TITLE
Failed to install `perseus-cli 0.3.0-beta.XX` with `clap 3.0.0-rc.X`

### ISSUE THREAD

**Issue by rieval** [arctic-hen7/perseus#85]
I tried to run the following command: 

`cargo install perseus-cli --version 0.3.0-beta.18`

It threw some similar errors: https://github.com/arctic-hen7/perseus/issues/65#issuecomment-945954443.

Then I noticed that `clap` had released the version `3.0.0-rc.X`, so I tried to editing `Cargo.toml` in the `perseus-cli-0.3.0-beta.18` folder.

```toml
[dependencies.clap]
version = "=3.0.0-beta.5" # <= pin the version
features = ["color"]
```

Finally I run the install command again and it runs successfully.

**Comment by arctic-hen7** [comment]
Ah, thanks for catching this one! I've just finished adding the new integration for Warp, so I'll release the fix for this in the next beta version, which should come out in a few hours.

**Comment by rieval** [comment]
Wow, thanks a lot. I will close this issue.

By the way, when do you plan to support `sycamore v0.7.0`? I'm looking for it. :)

**Comment by arctic-hen7** [comment]
No problem! Sycamore v0.7.0 will ship with the beta after next, which should be out by tomorrow, and then v0.3.0 *should* be able to go stable!

**Comment by rieval** [comment]
> No problem! Sycamore v0.7.0 will ship with the beta after next, which should be out by tomorrow, and then v0.3.0 _should_ be able to go stable!

Great work! 👍

**Comment by arctic-hen7** [comment]
@rieval just so you know what the heck is going on if you've tried to install Perseus again recently, there have been some minor critical failures in the two betas I just released, so we're now at version v0.3.0-beta.21, and the previous two have been yanked.

**Comment by rieval** [comment]
> @rieval just so you know what the heck is going on if you've tried to install Perseus again recently, there have been some minor critical failures in the two betas I just released, so we're now at version v0.3.0-beta.21, and the previous two have been yanked.

Wow, I tested it and it works fine.

Honestly, well done!

**Comment by arctic-hen7** [comment]
No problem, thank you very much!
